src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -1499,17 +1499,17 @@ static PyObject* NPySecObj_insert(NPySecObj* self, PyObject* args) {
         PyObject* tpyobj;
         if (PyArg_ParseTuple(args, "O", &tpyobj)) {
             auto _tpyobj_tracker = nb::borrow(tpyobj);
+            auto self = nb::borrow((PyObject*) raw_self);
```

Comment:
Because we don't know or prefer to pretend we don't know what happens in a `PyObject_CallMethod`, we assume it might reduce the ref count of `self` to zero.

To prevent this from happening we keep the reference count at `+1` by using `nb::borrow`.